roda-ui/roda-wui/src/main/java/org/roda/wui/api/controllers/BrowserHelper.java

```diff
@@ -387,6 +393,13 @@ public static BrowseRepresentationBundle retrieveBrowseRepresentationBundle(User
     } else {
       bundle.setRepresentationInformationFields(Collections.emptyList());
     }
+    ModelService model = RodaCoreFactory.getModelService();
+    try {
+      DistributedInstance distributedInstance = model.retrieveDistributedInstance(aip.getInstanceId());
+      bundle.setInstanceName(distributedInstance.getName());
+    } catch (NotFoundException e) {
+      LOGGER.warn("Could not retrieve the distributed instance", e);
+    }
```

Comment:
You need to check the instance type. The local instance does not have this configuration.

Reply:
Done